docs/docs/jsonnet/overview.md

```diff
@@ -10,7 +10,7 @@ menu: Writing Jsonnet
 expressing what shall be deployed to your Kubernetes cluster. Understanding
 Jsonnet is crucial to using Tanka effectively.
 
-This page covers the Jsonnet language itself. For more information on how to use Jsonnet with Kubernetes, see [the tutorial](/tutorial/jsonnet).
+This page covers the Jsonnet language itself. For more information on how to use Jsonnet with Kubernetes, see [the tutorial](/tutorial/jsonnet). For more information on Jsonnet itself, please have a look at [the official Jsonnet tutorial](https://jsonnet.org/learning/tutorial.html)
```

Comment:
This sentence is pretty duplicated .. maybe reword it a bit? Right now it reads like 

... Jsonnet language itself ... For more information ... For more information ... Jsonnet itself

Reply:
Yup, fixed in the most recent commit. PTAL.